apps/docs/src/pages/en/introduction.md

```diff
@@ -50,7 +50,7 @@ We will also make this clear in our library documentation as we see them.
 
 | Package                                                                                              | Release notes                                                                                     | Description                                              |
 | ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
-| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
+| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form/README.md)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
```

Comment:
this change to the `shields.io` URL from your latest commit is not right, it should not have `README.md`